.github/workflows/pr.yml

```diff
@@ -1,30 +1,28 @@
 # This GitHub Actions workflow automates the process of
 # publishing dataset collections to a staging environment
-# and creating a pull request (PR) in the veda-config repository
-# with the dataset configuration.
 # It is triggered by a pull request to the main branch
 # that modifies any files within the ingestion-data/dataset-config/ directory
 # The workflow includes steps to
 #   - publish the datasets,
-#   - create a PR in veda-config repository,
 #   - constantly updates the status of the workflow in the PR comment
 
-name: Publish collection to staging and create dataset config PR
+name: Publish collection to staging
 
 on:
   pull_request:
-    branches:
-      - main
+    branches: ['main', 'feat/automated-staging-publish']
     paths:
       # Run the workflow only if files inside this path are updated
-      - ingestion-data/dataset-config/*
+      # - ingestion-data/staging/dataset-config/*
+      - ingestion-data/testing/dataset-config/*
 
 jobs:
   dataset-publication-and-configuration:
     permissions:
       pull-requests: write
       contents: read
     runs-on: ubuntu-latest
+    environment: staging
```

Comment:
This is hardcoded for now to use the `staging` environment variable I defined in this github repo